src/test/ui/c-variadic/variadic-ffi-1.stderr

```diff
@@ -1,17 +1,17 @@
-error[E0045]: C-variadic function must have C or cdecl calling convention
+error[E0045]: C-variadic function must have a compatible calling convention, like C or cdecl
   --> $DIR/variadic-ffi-1.rs:9:5
    |
 LL |     fn printf(_: *const u8, ...);
-   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ C-variadics require C or cdecl calling convention
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ C-variadic function must have a compatible calling convention
```

Comment:
Nitpick: ideally we'd provide a structured suggestion that makes it explicit where and what to change:

```

error[E0045]: C-variadic function must have a compatible calling convention, like C or cdecl
  --> $DIR/variadic-ffi-1.rs:9:5
   |
LL |     fn printf(_: *const u8, ...);
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ C-variadic function must have a compatible calling convention
   |
help: declare `printf` with a compatible calling convention
   |
LL |   extern "C" {
   |           ~
LL |   extern "cdecl" {
   |           ~~~~~
LL |   extern "win64" {
   |           ~~~~~
LL |   extern "sysv64" {
   |           ~~~~~~
LL |   extern "efiapi" {
   |           ~~~~~~
```